operations/commit_queue.go

```diff
@@ -150,6 +151,45 @@ func mergeCommand() cli.Command {
 	}
 }
 
+func setModuleCommand() cli.Command {
+	return cli.Command{
+		Name:  "set-module",
+		Usage: "update or add module to an existing merge patch",
+		Flags: mergeFlagSlices(addLargeFlag(), addPatchIDFlag(), addModuleFlag(), addYesFlag(
+			cli.StringFlag{
+				Name:  joinFlagNames(refFlagName, "r"),
+				Usage: "merge branch `REF`",
+				Value: "HEAD",
+			},
+		)),
+		Before: mergeBeforeFuncs(
+			setPlainLogger,
+			requirePatchIDFlag,
+			requireModuleFlag,
+		),
+		Action: func(c *cli.Context) error {
+			params := moduleParams{
+				patchID:     c.String(patchIDFlagName),
+				module:      c.String(moduleFlagName),
+				ref:         c.String(refFlagName),
+				large:       c.Bool(largeFlagName),
+				skipConfirm: c.Bool(yesFlagName),
+			}
+
+			conf, err := NewClientSettings(c.Parent().String(confFlagName))
+			if err != nil {
+				return errors.Wrap(err, "problem loading configuration")
+			}
+			ac, rc, err := conf.getLegacyClients()
+			if err != nil {
+				return errors.Wrap(err, "problem accessing evergreen service")
+			}
+
+			return params.addModule(ac, rc)
```

Comment:
we should wrap this error?

Reply:
The thinking of not wrapping these errors is to pass the message from the server to the user unchanged. (I think @bsamek liked it this way)

Reply:
In these cases use WithStack()... the problem is that the errors you're passing around are local to your environment and only mirror things on the server side. 

Wrapping errors makes it possible to figure out what happened when you actually get an error and to be able to meaningfully understand where an error occurred. The only reason to not wrap an error is if you need to introspect the error, and if that's the case errors.Cause() unrwaps errors (and really error introspection is risky enough, that we should avoid it when possible, which makes the case for "not wrapping" to be minimal.